packages/govuk-frontend-review/package.json

```diff
@@ -51,5 +51,8 @@
     "slash": "^5.1.0",
     "supertest": "^6.3.3",
     "typedoc": "^0.24.7"
+  },
+  "optionalDependencies": {
+    "@types/marked": "^5.0.0"
```

Comment:
Thanks @domoscargin

Would you mind moving this one to the project **package.json** for now?

Seems to work more reliably, especially for dependencies of dependencies

We can move `@types/*` packages into relevant workspaces in future, but probably need [`typeRoots`](https://www.typescriptlang.org/tsconfig#typeRoots)